src/angular-lazy-component.js

```diff
@@ -78,6 +78,9 @@ class AngularLazyComponent extends React.Component {
       this.$injector.get('$rootScope').$destroy();
       this.$injector = null;
     }
+    if (this.manifest.unloadStylesOnDestroy === true) {
```

Comment:
For angular apps, it may be a default option, unless manifest creator specified `false`. What do you say ?